beacon-chain/blockchain/process_block.go

```diff
@@ -134,6 +135,9 @@ func (s *Service) onBlock(ctx context.Context, signed interfaces.ReadOnlySignedB
 	}
 	isValidPayload, err := s.notifyNewPayload(ctx, postStateVersion, postStateHeader, signed)
 	if err != nil {
+		if IsInvalidBlock(err) && InvalidBlockLVH(err) != [32]byte{} {
```

Comment:
Why don't we want to report when `LVH == [32]byte{}`? Isn't it still an invalid block?

Reply:
we do return with an error, but if the LVH is not available there's nothing we can do in forkchoice: the invalid block never made it to forkchoice  yet and there's no info to remove parents then